src/video/sdl_surface_ptr.hpp

```diff
@@ -81,6 +81,12 @@ class SDLSurfacePtr final
     SDL_FreeSurface(m_surface);
     m_surface = surface;
   }
+  void reset(SDLSurfacePtr& other)
```

Comment:
Add a new line before this.

Reply:
To preserve the initial tile with an ID of 0. I can add a comment for this.

Reply:
Is it needed, since it's an overload of the same function?

Reply:
I'd still think so. Those things irk me.